lmp-bsp.xml

```diff
@@ -6,7 +6,7 @@
   <default remote="lmp-mirrors" revision="master" sync-j="4"/>
 
   <project name="meta-arm" path="layers/meta-arm" revision="260e3adc2bf322f52d81c0642c825088a88bb051"/>
-  <project name="meta-freescale" path="layers/meta-freescale" revision="84be484645cdcc12bc12591874bfebd24cc0c9a6"/>
+  <project name="meta-freescale" path="layers/meta-freescale" revision="be9267fca85959c79b19ab65e48c8f3d7faf2b40"/>
```

Comment:
looks like the signed-of-by line is missing

Reply:
I was also trying to solve this problem but in the end everything is fine and it is a limitation of github https://github.com/foundriesio/lmp-manifest/pull/474#issuecomment-2603041206